cmd/neofs-node/config.go

```diff
@@ -553,8 +557,8 @@ func initCfg(appCfg *config.Config) *cfg {
 		scriptHash:          contractsconfig.Netmap(appCfg),
 		state:               netState,
 		workerPool:          netmapWorkerPool,
-		needBootstrap:       !relayOnly,
-		reBoostrapTurnedOff: atomic.NewBool(relayOnly),
+		needBootstrap:       !atomicRelayOnly.Load(),
```

Comment:
previous code with `relayOnly` looked much better to me than with `atomicRelayOnly.Load()`. U could change 2 strings instead of 6

Reply:
hope i got you right (not sure how that could be 2 vs 6) and fixed